modules/tide_content_collection/src/Plugin/Field/FieldWidget/ContentCollectionConfigurationWidget.php

```diff
@@ -0,0 +1,1201 @@
+
+    $configuration = $items[$delta]->configuration ?? [];
+
+    $element['#attached']['library'][] = 'field_group/formatter.horizontal_tabs';
+
+    $element['tabs'] = [
+      '#type' => 'horizontal_tabs',
+      '#tree' => TRUE,
+      '#weight' => 4,
+      '#group_name' => 'tabs',
+    ];
+
+    $this->buildContentTab($items, $delta, $element, $form, $form_state, $configuration, $json_object);
+    $this->buildLayoutTab($items, $delta, $element, $form, $form_state, $configuration, $json_object);
+    $this->buildAdvancedTab($items, $delta, $element, $form, $form_state, $configuration, $json_object);
+
+    return $element;
+  }
+
+  /**
+   * Build Content Tab.
+   *
+   * @param \Drupal\Core\Field\FieldItemListInterface $items
+   *   Field items.
+   * @param int $delta
+   *   The current delta.
+   * @param array $element
+   *   The element.
+   * @param array $form
+   *   The form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   * @param array $configuration
+   *   The YAML configuration of the listing.
+   * @param array $json_object
+   *   The json_object of the listing.
+   */
+  protected function buildContentTab(FieldItemListInterface $items, $delta, array &$element, array &$form, FormStateInterface $form_state, array $configuration = NULL, array $json_object = NULL) {
+    $settings = $this->getSettings();
+    $element['tabs']['content'] = [
+      '#type' => 'details',
+      '#open' => TRUE,
+      '#collapsible' => TRUE,
+      '#title' => $this->t('Content'),
+      '#group_name' => 'tabs_content',
+    ];
+
+    if ($this->indexHelper->isNodeTypeIndexed($this->index) && !empty($settings['content']['internal']['contentTypes']['enabled'])) {
+      $content_types_options = $this->indexHelper->getNodeTypes();
+      $allowed_content_types = array_filter($settings['content']['internal']['contentTypes']['allowed_values']);
+      if (!empty($allowed_content_types)) {
+        foreach ($content_types_options as $key => $value) {
+          if (!isset($allowed_content_types[$key])) {
+            unset($content_types_options[$key]);
+          }
+        }
+      }
+      $element['tabs']['content']['contentTypes'] = [
+        '#type' => 'checkboxes',
+        '#title' => $this->t('Select content types'),
+        '#options' => $content_types_options,
+        '#default_value' => $json_object['internal']['contentTypes'] ?? [],
+        '#weight' => 1,
+      ];
+    }
+
+    if ($this->indexHelper->isFieldTopicIndexed($this->index) && !empty($settings['content']['internal']['field_topic']['enabled'])) {
+      $default_values = $json_object['internal']['contentFields']['field_topic']['values'] ?? [];
+      $field_filter = $this->indexHelper->buildEntityReferenceFieldFilter($this->index, 'field_topic', $default_values);
+      if ($field_filter) {
+        $element['tabs']['content']['field_topic_wrapper'] = [
+          '#type' => 'details',
+          '#title' => 'Select topics',
```

Comment:
@joshua-salsadigital *all string literals must be translatable*

Reply:
All string literals are translatable